File: java/X42.java
public class X42 {
  public static void main(String[] args) {
    if (args.length > 0 && args[0].equals("42")) {
      System.err.println("It depends!");
      System.exit(42);
    }
    System.out.println("What is the meaning of life?");
    System.exit(0);
  }
}

File: python/x42.py
import sys

def main(args):
    """Implements X42"""
    if len(args) > 1 and args[1] == "42":
        sys.stderr.write("It depends!\n")
        sys.exit(42)
    sys.stdout.write("What is the meaning of life?\n")
    sys.exit(0)

if __name__ == "__main__":
    main(sys.argv)
